src/promptflow/tests/executor/e2etests/test_batch_engine.py

```diff
@@ -10,12 +10,16 @@
 import pytest
 
 from promptflow._constants import OUTPUT_FILE_NAME
+from promptflow._proxy._proxy_factory import ProxyFactory
```

Comment:
note this test should be put in promptflow-devkit test, instead of executor (promptflow-core) test